packages/react-instantsearch/examples/server-side-rendering/README.md

```diff
@@ -1,3 +1,12 @@
+# Server Side Rendering
+
+## How to use
```

Comment:
We could remove the title and use it as the first step of "to start the example". We can have a sentence like next.js =>  Download the example or clone the repo.

Also the title "The idea behind the example" doesn't feel right as the content doesn't correspond. Either we remove it, or we need to change the content. 

Otherwise I like the idea, but we should do it everywhere within this PR.